app/controllers/application_controller.rb

```diff
@@ -49,13 +49,11 @@ def filtered_records
       @records = @records.campaign(params[:campaign])
     end
 
-    # Count Second Opinions here for every record
+    # # Count Second Opinions here for every record
     @second_opinion_counts = RecordData.where(record: @records, opinion: true).group(:record_id).count
 
-    @records = @records.includes({ingest: :user}, :reviews)
-
     # Version ID, Version, or Feedback records are the only field we need to render on the home page, so just load that
-    @records = @records.includes(:record_datas).where(record_data: { column_name: ['VersionId', 'Entry_ID/Version', 'Version']}).or(@records.includes(:record_datas).where(record_data: {feedback:true})).references(:record_data)
+    @records = @records.joins(:record_datas).where("record_data.column_name IN (?, ?, ?) OR feedback = ?", 'VersionId', 'Entry_ID/Version', 'Version', true).distinct
```

Comment:
See major comment in pull request, this query gets at the essence of what the original programmer was after and includes feedback records, but I really think the query should just select all collection records instead.  The record class has a field called "recordable_type" which has only 2 values: collection or granule.